server/proxy/enable.go

```diff
@@ -51,13 +61,23 @@ func (p *Proxy) DisableApp(cc *apps.Context, app *apps.App) (md.MD, error) {
 		return "no change.", nil
 	}
 
+	cc, err = p.CleanUserCallContext(cc.ActingUserID, cc)
+	if err != nil {
+		return "", err
+	}
+
 	app.Disabled = true
-	resp := p.Call("", "", &apps.CallRequest{
-		Call:    *app.OnDisable,
-		Context: cc,
-	})
-	if resp.Type == apps.CallResponseTypeError {
-		p.mm.Log.Warn("OnDisable failed, app disabled anyway", "err", resp.Error(), "app_id", app.AppID)
+
+	var message md.MD
+	if app.OnDisable != nil {
```

Comment:
Should we fall back to a default call if none is provided similar to `OnInstall` or 		https://github.com/mattermost/mattermost-plugin-apps/blob/951f54b169dcda31aca45f71f189606036a5027e/server/proxy/oauth2.go#L70?

Reply:
@hanzei I honestly don't know. @levb Do you think we should notify the app even if it didn't declare that it wants to be notified on enable/disable?

Reply:
no